13_biblioteki/13_8_AlgorithmsBLAS.tex

```diff
@@ -0,0 +1,257 @@
+% !TEX encoding = UTF-8 Unicode
+\section{Algorytmy blokowe i BLAS}
+%%%%%%%%%%%%%%%%
+	\begin{frame}{Faktoryzacja}
+		Level 1, 2, 3 BLAS performance \\
+		Blocked algorithms \\
+		$$
+		\begin{pmatrix}
+		A_{11} & a_j & A_{13} \\
+		. & a_{jj} & a_j^T \\
+		. & . & A_{33}
+		\end{pmatrix}
+		=
+		\begin{pmatrix}
+		U_{11}^T & 0 & 0 \\
+		u_j^T & u_{jj} & 0 \\
+		U_{13}^T & u_j & U_{33}^T
+		\end{pmatrix}
+		*
+		\begin{pmatrix}
+		U_{11} & u_j  & U_{13} \\
+		0 & u_{jj} & u_j^T \\
+		0 & 0 & U_{33}
+		\end{pmatrix}
+		$$
+		Porównując współczynniki w kolumnie j otrzymujemy:
+		$
+		\begin{cases}
+		a_j = U_{jj}^T * u_j \\
+		a_{jj} = u_j^T * u_j + u_{jj}^2
+		\end{cases}
+		$ \\
+		Jeżeli $U_{11}$ został wyznaczony, to $u_j$ oraz $u_{jj}$ można obliczyć z:
+		$
+		\begin{cases}
+		U_{11}^T * u_j = a_j \\
+		u_{jj}^T = a_jj - u_j^T * u_{j}
+		\end{cases}
+		$
+	\end{frame}
+	\begin{frame}[fragile]{Realizacja z użyciem BLAS level 1}
```

Comment:
Sorry, but I'm missing something also here, what subsection? Shall I set the "Algorytmy blokowe i BLAS" as a frame title and make "Realizacja z użyciem BLAS level 1" a subsection title?